nixos/modules/system/boot/initrd-openvpn.nix

```diff
@@ -59,6 +59,13 @@ in
       cp -pv ${pkgs.glibc}/lib/libnss_dns.so.2 $out/lib
     '';
 
+    boot.initrd.systemd.storePaths = [
+      "${pkgs.openvpn}/bin/openvpn"
+      "${pkgs.iproute2}/bin/ip"
+      "${pkgs.glibc}/lib/libresolv.so.2"
+      "${pkgs.glibc}/lib/libnss_dns.so.2"
```

Comment:
I see the space-constrains, but having "ip" around for debugging might be useful for emergency shells?

And isn't `libnss_dns.so.2` required for things such as sshds UseDNS as well?

Reply:
About the first item: Why? I can trivially add it if I need it and it keeps the default size low.

About the second item: I see that but why not add it to the sshd module then? Or does ovpn use it as well?